cmake/lapack.cmake

```diff
@@ -132,7 +132,9 @@ set(CLASRC
   DEPRECATED/cgegs.f  DEPRECATED/cgegv.f  cgehd2.f cgehrd.f cgelq2.f cgelqf.f
   cgels.f  cgelsd.f cgelss.f DEPRECATED/cgelsx.f cgelsy.f cgeql2.f cgeqlf.f cgeqp3.f
   DEPRECATED/cgeqpf.f cgeqr2.f cgeqr2p.f cgeqrf.f cgeqrfp.f cgerfs.f
-  cgerq2.f cgerqf.f cgesc2.f cgesdd.f  cgesvd.f
+  cgerq2.f cgerqf.f cgesc2.f cgesdd.f  cgesvd.f cgejsv.f cgesvdx.f cgesvj.f
+  cgetrf2.f cgges3.f cggev3.f cgghd3.f cggsvd3.f cggsvp3.f cpotrf2.f csysv_aa.f
+  csytrs_aa.f

```

Comment:
@martin-frbg, `CLASRC` is outdated. There are more functions missing which leads to symbol errors. Do you update this list by hand or do you use a script?

Reply:
This is updated by hand, and cmake is still a second-class citizen here unfortunately. (The Makefile is correct however, so these should not be removed from exports/gensymbol)